app/src/main/res/values-es/strings.xml

```diff
@@ -92,6 +92,9 @@
     <string name="info_goal_no_transactions">Sin transacciones aún.</string>
     <string name="info_edit_transaction_button">Actualizar Transacción</string>
     <string name="info_transaction_onboarding_tip">Consejo: Desliza las transacciones hacia la izquierda o hacia la derecha para editarlas o eliminarlas.</string>
+    <string name="info_copy_button">Copy</string>
+    <string name="info_copy_icon_description">Copy Notes</string>
+    <string name="info_copy_alert">Copied</string>
```

Comment:
I think something like "Copied Successfully!" would look much better. What do you think?

Reply:
Yeah, sounds good to me, gonna add that change in the next commits